src/batch.rs

```diff
@@ -446,6 +457,24 @@ fn ddbjson_val_to_attrval(ddb_jsonval: &JsonValue) -> Option<AttributeValue> {
             ns: Some(set_logic(x)),
             ..Default::default()
         })
+    } else if let Some(x) = ddb_jsonval.get("BS") {
+        let binary_set = x
+            .as_array()
+            .expect("should be valid JSON array")
+            .iter()
+            .map(|el| {
+                Bytes::from(
+                    general_purpose::STANDARD
+                        .decode(el.as_str().unwrap())
+                        .expect("binary inputs should be base64 encoded"),
+                )
```

Comment:
This code can be extracted as a function and can be deduplicated.

Reply:
I have implemented the changes